mesh/mesh.cpp

```diff
@@ -5250,6 +5257,181 @@ void Mesh::LoadPatchTopo(std::istream &input, Array<int> &edge_to_knot)
 
    FinalizeTopology();
    CheckBdrElementOrientation(); // check and fix boundary element orientation
+
+   // Generate knot 2 edge mapping -- if not specified
+   if (edge_to_knot.Size() == 0)
+   {
+      edge_vertex = new Table(NumOfEdges, 2);
+      edge_to_knot.SetSize(NumOfEdges);
+      int notset = -9999999;
```

Comment:
How about `constexpr int notset = -1;`?

Reply:
Hi @dylan-copeland, good suggestion! I have implemented the constexpr. Setting it to -1 is will not work as I remember that v1 and v0 can be negative as well for orientation purposes. So -1 would be the "flipped" version of 0.